### DISCUSSION TITLE
How to change direction? (direction="LR" does not work...)

### DISCUSSION THREAD

**Discussion by asannikov** [mingrammer/diagrams#867] [Q&A]
I have this code:
```
from diagrams import Cluster, Diagram, Edge, Node
from diagrams.programming.flowchart import Action
from diagrams.aws.devtools import Codepipeline

graph_attr={"style": "solid", "margin": "10", "padding": "0"}

# LR (left-to-right)
# RL (right-to-left)
# TB (top-down)
# BT (bottom-up)
with Diagram("workflow", show=False, filename="workflow",  direction="LR", outformat=["png"]):
    with Cluster("Local machine", graph_attr=graph_attr):
        Action("Build", fontsize="10", fixedsize="true", width="0.6", height="0.6", pos="nw")
        Action("Test", fontsize="10", fixedsize="true", width="0.6", height="0.6", pos="nw")

    with Cluster("AWS", graph_attr=graph_attr):
        pipeline1 = Codepipeline('pipleine')
        pipeline2 = Codepipeline('pipleine')
        pipeline3 = Codepipeline('pipleine')
```

and it's supposed to be in horizontal orientation. But it's generated vertically:
![workflow](https://user-images.githubusercontent.com/29725912/226095686-f30894dd-693a-4fd3-90cb-5f7a01cd1731.png)

how to force "LR" then?
and another question: if i want to have one vertical schema and one horizontal in one diagram. Is it possible to implement?

**Comment by GorgiAstro** [comment]
Same question here, I have the same issue, I can't manage to have an horizontal layout, even when using the `direction="LR"` argument.

**Comment by clayms** [comment]
you have to connect your nodes with `-` , `>>`, or `<<`.   Otherwise, graphviz will just stack the nodes at the same level.
So, `pipeline1 - pipeline2`
